lib/inspec_tools/inspec.rb

```diff
@@ -321,5 +296,87 @@ def populate_groups
       end
       @benchmark.group = group_array
     end
+
+    def generate_title(title, json, date)
+      title ||= "Untitled - Checklist Created from Automated InSpec Results JSON; Profiles: #{json['profiles'].map { |x| x['name'] }.join(' | ')}"
+      title + " Checklist Date: #{date || Date.today.to_s}"
+    end
+
+    def handle_vuln_num(stig_data_list, control)
+      return stig_data_list if control[:vuln_num].nil?
+
+      stig_data_list.push(HappyMapperTools::StigChecklist::StigData.new('Vuln_Num', control[:vuln_num]))
+    end
+
+    def handle_severity(stig_data_list, control)
+      return stig_data_list if control[:impact].nil?
+
+      value = Utils::InspecUtil.get_impact_string(control[:impact])
+      return stig_data_list if value == 'none'
+
+      value = 'high' if value == 'critical'
```

Comment:
We don't use critical anywhere in the application, and the other spot that get_impact_string is called also remaps critical to high. Why don't we just fix the `get_impact_string` function? (Or more specifically fix IMPACT_SCORES since it is only used by get_impact_string).

Reply:
We should be using `get_impact_string` method more when we work on #107